### ISSUE TITLE
Website Accessiblity

### ISSUE THREAD

**Issue by KCollins** [HamSCI/psws-charette#26]
# Website Accessibility
## Description
Ensure that website/annotation system are accessible for screen readers, etc. (Consult with Scranton disability office, Lackawanna Blind Assoc. to verify specs)

### Suggested by:
Kristina Collins, KD8OXT, kvc2@case.edu

### Additional comments:
nan

**Comment by KCollins** [comment]
Relates to #22 . @w2naf , this isn't something we have resources to do right now, but might be good down the line. I just want to get it on your radar.
